packages/riipen-ui/src/components/AccordionDetails.test.jsx

```diff
@@ -0,0 +1,63 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import AccordionDetails from "./AccordionDetails";
+
+describe("AccordianDetails", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<AccordionDetails />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("snapshot test", () => {
+    const wrapper = mount(<AccordionDetails />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("displays children", () => {
+      const child = <h5>Accordian Details</h5>;
+      const wrapper = mount(<AccordionDetails children={child} />);
+
+      expect(wrapper.contains(child)).toEqual(true);
+    });
+  });
+
```

Comment:
@imogenem check out this `children` test. It's a good example of what I was talking about in your PR.